src/Spatial/Euclidean/Circle3D.cs

```diff
@@ -15,5 +15,16 @@ public Circle3D(Point3D centerPoint, UnitVector3D axis, double radius)
             this.Axis = axis;
             this.Radius = radius;
         }
+
+        public Circle3D(Point3D p1, Point3D p2, UnitVector3D axis)
+        {
+            this.CenterPoint = Point3D.MidPoint(p1, p2);
+            this.Axis = axis;
+            this.Radius = p1.DistanceTo(CenterPoint);
+        }
```

Comment:
Hmm, maybe I was too quick at merging, don't really like this ctor as it is not very intuitive.